### PR TITLE
WIP: Check for metadata

### PR DESCRIPTION
**Pull request by BhargavBhandari90**
For https://github.com/wp-cli/doctor-command/issues/94

### CODE REVIEW

**Comment by BhargavBhandari90** [comment]
@danielbachhuber 
There are already two meta tags available on fresh WP setup which are
```
<meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="generator" content="WordPress 4.8.4" />
```

So I think we have to ignore these and find other that these two meta tags right?
Because if we check for any one meta tag availability , then it will always return true.

File: inc/checks/class-check-meta-tags.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ */
+class Check_Meta_Tags extends Plugin {
+
+	// Meta tags reg. ex.
+	private $metatags_expression = "/<meta[^>]+(?:name|property)=\"([^\"]*)\"[^>]+content=\"([^\"]*)\"[^>]*>/";
+
+	public function run() {
+
+		// Fetch homepage resposne.
+		$response = wp_remote_get(site_url());
```
Comment (danielbachhuber) [inline inc/checks/class-check-meta-tags.php]: Rather than make a remote request, we should instead render and parse the template.

**Comment by danielbachhuber** [comment]
> So I think we have to ignore these and find other than these two meta tags right?
> Because if we check for any one meta tag availability , then it will always return true.

Sorry, to clarify, the end user should be able to define which meta tags they expect to be present.

**Comment by danielbachhuber** [comment]
@BhargavBhandari90 Planning to pick this back up?

**Comment by BhargavBhandari90** [comment]
Yes @danielbachhuber. I was little busy last weekend. I'll try to provide solution within this week.

**Comment by danielbachhuber** [comment]
Closing per lack of activity.

**Pull request closed without merging**